### DISCUSSION TITLE
Spill volumes on k8s

### DISCUSSION THREAD

**Discussion by igorvoltaic** [trinodb/trino#22467] [Q&A] [answered]
Hey Folks, anyone used Trino on k8s? How do you configure spills? Is it a good idea to use emptyDir. I’m a bit confused since docs say do not use same disk where JVM runs? Or is it better to use persistentVolume for spilling?

**Comment by nineinchnick** [comment, marked as answer]
The same question was asked on Slack, let me copy some responses here if anyone else stumbles upon this:

@nineinchnick said:
> `emptyDir` is using memory, so it depends on how much margin you have on the node, and how big spills do you expect - it might get expensive.
Using a PV sounds fine.
The docs focus on bare metal installations, that’s why it mentions a separate disk. It all comes down to IO and what happens when spills fill up all the available space. 

@Pluies said:
> We run spill-to-disk on EBS-backed gp3 PVCs here. It works well, though you have to monitor io etc so that it doesn’t become too much of a bottleneck
